test/cbmc/proofs/MQTT_Connect/Makefile

```diff
@@ -92,4 +92,6 @@ PROJECT_SOURCES += $(SRCDIR)/source/core_mqtt.c
 PROJECT_SOURCES += $(SRCDIR)/source/core_mqtt_serializer.c
 PROJECT_SOURCES += $(SRCDIR)/source/core_mqtt_state.c
 
+EXPENSIVE = true
```

Comment:
Just curious as to what exactly this does? How do you know what proofs should be marked as `EXPENSIVE`?

Reply:
Setting EXPENSIVE to those particular proofs allows those proofs to be run separately (each one on their own) whereas the rest run in the usual parallelized fashion.
MQTT_ProcessLoop was EXPENSIVE as seen [**HERE**](https://d2s5j4r0gkya4b.cloudfront.net/final/4c742b5e-6010-4ac3-843f-6133e0158be6/pipelines/MQTT_ProcessLoop/index.html#job-c820f83e-14ad-4663-9f88-6ead71deaaab).
I did the same for two other proofs which were taking longer as seen on the runtime graph on [**THIS**](https://d2s5j4r0gkya4b.cloudfront.net/final/4c742b5e-6010-4ac3-843f-6133e0158be6/index.html) page.

Reply:
Thanks @ronakfof for the thorough explanation. I'll also let each repository owner check the memory usage graphs so that they can appropriately mark certain proofs as `EXPENSIVE`.